### PR TITLE
ISSUE #3822 - UI fixes

### LINKED ISSUE #3822: UI fixes
### Tasks
- [ ]  Arrows on custom ticket detail are way too big

Build
image

Design
image

- [ ] Missing Safetibase icon inside Safetibase cards

image

- [ ] Gaps between action buttons at the top of cards

image

- [ ]  Instant clash tab on compare tab should be on a single line

Build
image

### PR DESCRIPTION
**Pull request by The-Daniel**
This fixes #3822 

#### Description
- Fix size and position of chevrons when cycling between ticket details
- Add missing safetibase icon in safetibase card
- If a card action (buttons in top right of viewer card) is missing then there is no empty space
- Instant clash tab on compare tab is single line
- Changed hex value for BASE_LIGHTEST to match figma theme
- All custom ticket properties have the same border colour
- Tweak input styles to match figma more closely
- Error modal has a max-width
- Hovering over additional assignees circle lists ALL of the assignees, not just the overflow
- Reduced back arrow size in custom tickets
- when creating a new ticket the padding in the save ticket button is equal
- Added a space after each comma in the assignees input

#### Test cases
Ensure other inputs in the viewer do not have messed up styling

### CODE REVIEW

**Comment by Amantini1997** [comment]
I'm not sure about this, but should the `0` be displayed?
image

**Comment by The-Daniel** [comment]
> I'm not sure about this, but should the `0` be displayed? image

Fixed with merge

**Review by sanmont3drepo — DISMISSED** [review]

**Comment by george-hadfield** [comment]
"If a card action (buttons in top right of viewer card) is missing then there is no empty space"

Not sure if this is exactly what you meant but there's an unwanted empty space inside a safetibase risk as shown in screenshot here.. 

image

**Comment by george-hadfield** [comment]
"Reduced back arrow size in custom tickets"

Size is fine but the alignment is off. Can the arrow please be positioned in the same position as the custom ticket icon one tier up. As when you flick between the list and the ticket detail, there's a noticeable gap to the left of the arrow. Please ensure text moves to the left with the arrow.

image

**Comment by george-hadfield** [comment]
### Padding or margin is too big on assignees tooltip

Please can make these values the same for assignees tooltip and new ticket options

If you click 'New ticket' you will see the correct spacing as shown below
image

Compare that to the assignees tooltip and you'll notice the excessive left spacing shown below
image

**Comment by george-hadfield** [comment]
### Remove background hover state on custom ticket list

image

Please keep the pointer on hover though and the underline when hovering the title. This request is just for the backgrond.

**Review by george-hadfield — CHANGES_REQUESTED** [review]
A few small tweaks

**Comment by The-Daniel** [comment]
@george-hadfield I've noticed that selected buttons in the bottom toolbar use a color from the V4 palette.
image
Would you like me to change this to something like FAVOURITE_MAIN?

**Comment by george-hadfield** [comment]
> @george-hadfield I've noticed that selected buttons in the bottom toolbar use a color from the V4 palette. image Would you like me to change this to something like FAVOURITE_MAIN?

Please use primary.main instead

**Review by george-hadfield — CHANGES_REQUESTED** [review]
A few small tweaks then all good!

**Review by george-hadfield — DISMISSED** [review]

File: frontend/src/v5/ui/v4Adapter/overrides/cards/compare.overrides.ts [hunk 1]
```diff
@@ -15,12 +15,17 @@
...
 
+import { Tabs } from '@/v4/routes/viewerGui/components/compare/compare.styles';
 import { Revisions } from '@/v4/routes/viewerGui/components/compare/components/compareDiffItem/compareDiffItem.styles';
 import { FilterPanel } from '@/v4/routes/viewerGui/components/compare/components/compareFilters/compareFilters.styles';
 import { SelectField } from '@/v4/routes/viewerGui/components/compare/components/revisionsSelect/revisionsSelect.styles';
 import { css } from 'styled-components';
 
 export default css`
+	${Tabs} .MuiTab-root {
+		white-space: nowrap;
```
Comment (Amantini1997) [inline frontend/src/v5/ui/v4Adapter/overrides/cards/compare.overrides.ts:26]: I think might as well be added to the general theme

File: frontend/src/v5/ui/v4Adapter/overrides/cards/safetiBase/safetiBase.overrides.ts [hunk 2]
```diff
@@ -37,6 +37,10 @@ export default css`
 		padding: 0 15px;
 		/* TODO - fix after new palette is released */
 		background-color: #F7F8FA;
+
+		.MuiSelect-select {
+			border: 1px solid ${({ theme }) => theme.palette.base.lightest};
```
Comment (Amantini1997) [inline frontend/src/v5/ui/v4Adapter/overrides/cards/safetiBase/safetiBase.overrides.ts:42]: Same comment as above 
image

File: frontend/src/v5/ui/v4Adapter/overrides/cards/issues/properties.overrides.ts [hunk 3]
```diff
@@ -32,6 +32,10 @@ export default css`
 				padding: 0px 11px;
 			}
 
+			.MuiSelect-select {
+				border: 1px solid ${({ theme }) => theme.palette.base.lightest};
```
Comment (Amantini1997) [inline frontend/src/v5/ui/v4Adapter/overrides/cards/issues/properties.overrides.ts:36]: This one also override the border-color when the dropdown is selected 
image

**Review by Amantini1997 — CHANGES_REQUESTED** [review]

**Review by Amantini1997 — APPROVED** [review]

**Review by sanmont3drepo — APPROVED** [review]

**Pull request merged**